Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in we_ship_client.gemspec
 gemspec
 
-gem 'loogi_http', '~> 1.0'
+gem 'loogi_http', '~> 2.0', github: 'JuulLabs-OSS/loogi_http', branch: 'jdutil/faraday'
```

Comment:
is this intentional?

Reply:
@earthtosid yes for now I wanted to test that my changes on the loogi_http PR passed here. Once I merge the loogi_http PR and it pushes out a new version of the gem I'll update this to refer to that published gem version.

Reply:
Updated